packages/web/src/pages/sign-up-page/components/SignUpWithMetaMaskButton.tsx

```diff
@@ -1,58 +1,23 @@
-import { useContext, useState } from 'react'
+import { useState } from 'react'
 
-import { signUpFetch } from '@audius/common/api'
-import { AudiusQueryContext } from '@audius/common/audius-query'
 import { Button, IconMetamask } from '@audius/harmony'
-import { useFormikContext } from 'formik'
-import { useDispatch } from 'react-redux'
 
-import { setValueField } from 'common/store/pages/signon/actions'
 import { useNavigateToPage } from 'hooks/useNavigateToPage'
 import ConnectedMetaMaskModal from 'pages/sign-up-page/components/ConnectedMetaMaskModal'
-import { SIGN_IN_PAGE, SIGN_UP_HANDLE_PAGE } from 'utils/route'
-
-import { SignUpEmailValues } from '../pages/CreateEmailPage'
+import { SIGN_UP_HANDLE_PAGE } from 'utils/route'
 
 const messages = {
   signUpMetamask: 'Sign Up With MetaMask',
   unknownError: 'Unknown error occurred.'
 }
 export const SignUpWithMetaMaskButton = () => {
-  const queryContext = useContext(AudiusQueryContext)
-  const dispatch = useDispatch()
   const [isMetaMaskModalOpen, setIsMetaMaskModalOpen] = useState(false)
-  const { setErrors, validateForm, values } =
-    useFormikContext<SignUpEmailValues>()
   const navigate = useNavigateToPage()
   const handleSuccess = () => {
     navigate(SIGN_UP_HANDLE_PAGE)
   }
-  const handleClick = async () => {
```

Comment:
whats the context for the change here?

Reply:
this is technically related to a different bug where the modal wasn't popping up